app/org/sagebionetworks/bridge/DefaultStudyBootstrapper.java

```diff
@@ -1,48 +1,35 @@
 package org.sagebionetworks.bridge;
 
-import java.util.Set;
-
 import org.sagebionetworks.bridge.dynamodb.DynamoStudy;
 import org.sagebionetworks.bridge.exceptions.EntityNotFoundException;
+import org.sagebionetworks.bridge.models.studies.PasswordPolicy;
 import org.sagebionetworks.bridge.models.studies.Study;
-import org.sagebionetworks.bridge.services.StudyConsentService;
 import org.sagebionetworks.bridge.services.StudyService;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
-import com.google.common.collect.Sets;
-
-@Component("studyConsentBootstrapper")
-public class StudyConsentBootstrapper {
-    
-    private static Logger logger = LoggerFactory.getLogger(StudyConsentBootstrapper.class);
+@Component("defaultStudyBootstrapper")
+public class DefaultStudyBootstrapper {
     
     @Autowired
-    public StudyConsentBootstrapper(StudyService studyService, StudyConsentService studyConsentService) {
+    public DefaultStudyBootstrapper(StudyService studyService) {
         try {
-            Study study = studyService.getStudy("api");
-            try {
-                Set<String> atts = Sets.newHashSet("phone", "can_be_recontacted");
-                if (study.getUserProfileAttributes().size() != atts.size()) {
-                    study.setUserProfileAttributes(atts);
-                    studyService.updateStudy(study);
-                }
-            } catch(Exception e) {
-                logger.warn(e.getMessage(), e);
-            }
+            studyService.getStudy("api");
         } catch(EntityNotFoundException e) {
             Study study = new DynamoStudy();
             study.setName("Test Study");
             study.setIdentifier("api");
+            study.setSponsorName("Sage Bionetworks");
             study.setMinAgeOfConsent(18);
             study.setResearcherRole("api_researcher");
             study.setConsentNotificationEmail("bridge-testing+consent@sagebridge.org");
+            study.setTechnicalEmail("bridge-testing+technical@sagebridge.org");
+            study.setSupportEmail("bridge-testing+support@sagebridge.org");
```

Comment:
This still says sagebridge.org instead of sagebase.org.

Also, the email in SES is support@sagebridge.org, which is what we want for the sender (support email).